(GH-1790) Wrap Sensitive plan parameters as Sensitive wrapper type (#1823)
puppetlabs/bolt: documentation/writing_plans.md

- [Task metadata types](writing_tasks.md#)

### Sensitive parameters

> Parameters that are sensitive and should not be displayed in logs can be masked using the `Sensitive` data type.

Reviewer: ```suggestion
Use the `Sensitive` data type to mask parameters that should not be displayed in logs.
```

> When passing a value to a `Sensitive` parameter, Bolt will automatically mask the value before the plan is run.

Reviewer: ```suggestion
When you pass a value to a `Sensitive` parameter, Bolt automatically masks the value before the plan is run.
```

> The unmasked value can be accessed by calling the `unwrap` function on the parameter.

Reviewer: ```suggestion
To access the unmasked value, call the `unwrap` function on the parameter.
```

> Use the `Sensitive` data type to mask parameters that should not be displayed in logs can be masked using the 
> `Sensitive` data type. When you pass a value to a `Sensitive` parameter, Bolt automatically masks the value
> before the plan is run. To accewss the unmasked value, call the `unwrap` function on the parameter.

Reviewer: ```suggestion
Use the `Sensitive` data type to mask parameters that should not be displayed in logs. 

When you pass a value to a `Sensitive` parameter, Bolt automatically masks the value
before the plan is run. 

To access the unmasked value, call the `unwrap` function on the parameter.
```
